lib/Bailador/Request.pm

```diff
@@ -25,40 +27,61 @@ class Bailador::Request {
     }
     multi method params ($source) {
         my %ret;
+        my $source_data = '';
```

Comment:
Ok. I was thinking `my Str $source_data` was enough. So you're right.

Btw, thanks for this PR ! :)